### PR TITLE
Remove legacy routing code

### PR DESCRIPTION
**Pull request by Hyperkid123**
Fixes initialization issues in the latest chrome release.

More details in slack conversation

@himdel @ZitaNemeckova commit `185e202bd2a45e2754b2e55c615bf03ca256fc5a` is the one that fixes the issue and should be cherry-picked all the way to prod-stable build. The other one is just some clean-up.

### CODE REVIEW

File: src/loaders/insights/Routes.js [hunk 1]
```diff
@@ -153,150 +122,62 @@ InsightsRoute.propTypes = {
...
-          some(Paths, (p) => p === path) ? null : (
-            <Redirect push to={Paths.notFound} />
-          )
-        }
-      />
-    </Switch>
+    /**
+     * TODO: Add global loader component to the fallback
+     */
+    <Suspense fallback={null}>
```
Comment (himdel) [inline src/loaders/insights/Routes.js:130]: shouldn't we still fallback to `LoadingPageWithHeader`?

Comment (Hyperkid123) [reply src/loaders/insights/Routes.js:130]: @himdel if that is your loader then yes.

Comment (himdel) [reply src/loaders/insights/Routes.js:130]: yeah, that's the one hardcoded in the old asyncComponent :)

**Comment by himdel** [comment]
(cypress failure unrelated, pr-checks is failing on `src/loaders/insights/Routes.js` vs prettier)

**Comment by himdel** [comment]
Cc @ZitaNemeckova any chance your insights mode is working? 

I think this PR is ready, would approve and merge :+1: ... but when testing, `qa.foo`, `ci.foo` and `stage.foo` come completely blank, while `prod.foo` has the menu and top nav while only the automation hub parts are blank :) .. and I get the same for master as well.

**Comment by ZitaNemeckova** [comment]
@himdel Insights mode gives me 403 on every page :/

**Comment by himdel** [comment]
Yeah, agreed, we'll need to fix the local setup, but not blocking this PR for it :) We can test on stage.

**Pull request merged**